runtime/j9vm/j9vmnatives.xml

```diff
@@ -391,4 +391,9 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		<export name="JVM_ReportFinalizationComplete"/>
 	</exports>
 
+	<exports group="jdk19">
```

Comment:
This new group needs to be added to the module.xml as well.

Reply:
nm, I see it now.

Reply:
It is in `j9vm/module.xml` & `redirector/module.xml`.
Assuming that's correct since the PR was approved.